packages/patch-events/src/interfaces/event-effect.ts

```
@@ -0,0 +1,4 @@
+import { EventEffectParams } from './event-effect-params.js';
+
+export type EventEffectDestructor = () => void;
+export type EventEffect = (params: EventEffectParams) => void | EventEffectDestructor;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)